integration-test/596-add-hiking-routes.py

```diff
@@ -93,10 +124,25 @@ def test_driveway_nwn(self):
         # Dogbane - service=driveway - part of American Discovery Trail
         # (nwn)
         # should be visible at z11
-        self.load_fixtures([
-            'http://www.openstreetmap.org/way/16000421',
-            'http://www.openstreetmap.org/relation/1544944',
-        ], clip=self.tile_bbox(11, 491, 762))
+        self.generate_fixtures(
+            dsl.way(16000421, wkt_loads('LineString (-93.68584719999999777 41.74244529999999997, -93.6859489999999937 41.74249919999999747, -93.68607550000000117 41.74251509999999854, -93.68619499999999789 41.74251499999999737, -93.68646250000000464 41.74245650000000296, -93.68669930000000079 41.74234510000000142, -93.68697509999999795 41.7422196999999997, -93.6872615000000053 41.74211700000000036, -93.68774890000000255 41.7419809000000015)'), {
```

Comment:
Try a simple `tile_diagnal` and `relation` dsl features inside a given tile, then testing again. (Versus figuring out what tile to test for for that geometry.)

Reply:
Inside tile `11, 491, 762` (what you're testing below) or a child tile.